app/Http/Controllers/Webapp/PostController.php

```diff
@@ -0,0 +1,1890 @@
+        }
+
+        if($post->isLock($this->currentSession)) {
+            $this->_flashMessage($request, 'このコンテンツは現在ロックされています。下書きに保存できません。', 'danger');
+            $this->activityLog->setInfo('Locked');
+            if(!$request->get('selected_ids'))
+                return redirect()->to(action('Webapp\\PostController@getIndex'))->send();
+            else
+                return;
+        }
+        if($post->site_id == ''){
+            $this->_flashMessage($request, 'ターゲットサイトを選んでください', 'danger');
+            $this->activityLog->setInfo('Missing target stie');
+            return redirect()->to(action('Webapp\\PostController@getIndex'))->send();
+        }
+
+        if($post->published_status == 0) {
+            $this->_flashMessage($request, '下書きに保存しました。', 'warning');
+            $this->activityLog->setInfo('Already draft');
+            if(!$request->get('selected_ids'))
+                return redirect()->to(action('Webapp\\PostController@getIndex'))->send();
+            else
+                return;
+        }
+
+        $this->activityLog->addMeta('PreviousPublishedStatus', $post->published_status);
+
+        try {
+            $apiUrl = Site::find($post->site_id)->api_url;
+            if($post->target_post_id > 0) {
+                $returnData = (new CdsApiDriver($apiUrl, [
+                    'content' => $post->content,
+                    'title' => $post->title,
+                    'categories' => [$post->category],
+                    'post_excerpt' => $post->description,
+                    'post_id' => $post->target_post_id,
+                    'post_status' => 'draft',
+                    'feature_img' => $post->feature_img,
+                    'author_name' => $post->author_name,
+                    'cback_url' => action('Webapp\\PostController@getEdit', $post->id),
+                    'priority' => 0
+                ]))->send('edit')->getResult();
+                $post->content = Post::replaceIDButtonLink($returnData->data->post_id, $post->content, $post->target_post_id);
+                $post->target_post_id = $returnData->data->post_id;
+            }
+
+            $post->published_status = 0;
+            $this->_flashMessage($request, 'OK', 'success');
+        } catch(\Exception $e) {
+            $post->content = Post::replaceIDButtonLink('__POSTID__', $post->content, $post->target_post_id);
+            $this->activityLog->setInfo('Error when reaching to target site');
+            $this->activityLog->addMeta('FailedPostToTargetSite', 'error when reaching to target site');
+            $this->_flashMessage($request, '更新に失敗しました。もう一度ご確認ください。', 'warning');
+        }
+
+        $post->save();
+
+        if($request->get('hasFilter', false) !== false) {
+            return redirect()->back()->send();
+        }
+
+
+        $this->activityLog->addMeta('NewPublishedStatus', $post->published_status);
+
+        if(!$request->get('selected_ids'))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use curly braces or indentation to denote the code conditionally executed by this "if". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS3973)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS121)